openpype/hosts/blender/plugins/create/create_workfile.py

```diff
@@ -66,32 +66,36 @@ def create(self):
                     asset_doc,
                     project_name,
                     host_name,
-                    existing_instance,
+                    workfile_instance,
                 )
             )
             self.log.info("Auto-creating workfile instance...")
-            current_instance = CreatedInstance(
+            workfile_instance = CreatedInstance(
                 self.family, subset_name, data, self
             )
-            instance_node = bpy.data.collections.get(AVALON_CONTAINERS, {})
-            current_instance.transient_data["instance_node"] = instance_node
-            self._add_instance_to_context(current_instance)
+            self._add_instance_to_context(workfile_instance)
+
         elif (
             existing_asset_name != asset_name
-            or existing_instance["task"] != task_name
+            or workfile_instance["task"] != task_name
         ):
             # Update instance context if it's different
             asset_doc = get_asset_by_name(project_name, asset_name)
             subset_name = self.get_subset_name(
                 task_name, task_name, asset_doc, project_name, host_name
             )
             if AYON_SERVER_ENABLED:
-                existing_instance["folderPath"] = asset_name
+                workfile_instance["folderPath"] = asset_name
             else:
-                existing_instance["asset"] = asset_name
+                workfile_instance["asset"] = asset_name
+
+            workfile_instance["task"] = task_name
+            workfile_instance["subset"] = subset_name
 
-            existing_instance["task"] = task_name
-            existing_instance["subset"] = subset_name
+        instance_node = bpy.data.collections.get(AVALON_CONTAINERS)
```

Comment:
I think the choice for `AVALON_CONTAINERS` was originally mine - however, looking at it now I feel like that should've been `AVALON_INSTANCES` instead since containers are for loaded content and instances for created instances.

@simonebarbieri thoughts?

Reply:
Mmmh yes, that's a good point actually, maybe it would be better in `AVALON_INSTANCES`. 
But it's a bit out of scope for this PR, so this can be addressed in another PR, maybe trying to handling the breaking change? :)